### ISSUE TITLE
Impasse 1.2.1 on Redmine 1.4.X TemplateError undefined method button_tag

### ISSUE THREAD

**Issue by Salivaxiu** [kawasima/redmine_impasse#122]
Processing ImpasseTestCaseController#index (for 172.18.0.18 at 2012-09-11 11:05:02) [GET]
  Parameters: {"project_id"=>"f", "action"=>"index", "controller"=>"impasse_test_case"}
Rendering template within layouts/base
Rendering impasse_test_case/index
DEPRECATION WARNING: ApplicationHelper#labelled_tabular_form_for is deprecated and will be removed in Redmine 1.5. Use #labelled_form_for instead. (called from labelled_tabular_form_for at D:/redmine/app/helpers/application_helper.rb:921)

ActionView::TemplateError (undefined method `button_tag' for #ActionView::Base:0xd4aa998) on line #115 of vendor/plugins/redmine_impasse/app/views/impasse_test_case/index.html.erb:
112:           <%= output %>
113:         <% end %>
114:    <p>
115:      <%= button_tag l(:button_copy), :id => 'button-copy-exec' %>
116:      <%= button_tag l(:button_cancel), :id => 'button-copy-cancel' %>
117:    </p>
118:       <% else %>

```
vendor/plugins/redmine_impasse/app/views/impasse_test_case/index.html.erb:115
```

Rendering D:/redmine/public/500.html (500 Internal Server Error)

**Comment by speedy32129** [comment]
Same on Redmine 1.2.3 however please note I downloaded impasse 1.2.1 and the init.rb says 1.2.0

**Comment by feicipet** [comment]
Same issue on 1.4.3 on Ubuntu. Impasse installed using git clone as per Administration instructions. Thanks.

**Comment by adepier** [comment]
same issue in redmine 2.1

**Comment by feicipet** [comment]
OK, I think I've found the issue:

According to http://apidock.com/rails/v3.1.0/ActionView/Helpers/FormTagHelper/button_tag the tag "button_tag" only exists in Ruby on Rails 3.1.0 onwards. I'm running Redmine on top of RoR 2.3.14, which is the stock version of Rails that comes with Ubuntu 12.04. Looks like I need to upgrade Rails.

**Comment by kawasima** [comment]
I fixed it at version 1.2.2. Thank you for your infomation.
